package.json

```diff
@@ -95,7 +95,8 @@
     "fitness-functions": "ts-node development/fitness-functions/index.ts",
     "generate-beta-commit": "node ./development/generate-beta-commit.js",
     "validate-branch-name": "validate-branch-name",
-    "label-prs": "ts-node ./.github/scripts/label-prs.ts"
+    "label-prs": "ts-node ./.github/scripts/label-prs.ts",
+    "add_release_label_to_pr_and_linked_issues": "ts-node ./.github/scripts/add_release_label_to_pr_and_linked_issues.ts"
```

Comment:
Nit: conventionally these scripts use` kebak-case` rather than `snake_case`

Reply:
This is now fixed with this [commit](https://github.com/MetaMask/metamask-extension/pull/19061/commits/52c9f9f510d3580764bb74cb2ddce7383f634c2a).